index.html

```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
 			<dd>
-			  <p><a class="termref" data-lt="attribute">Attributes</a> that are essential to the nature of a given <a>object</a>, or that represent a data value associated with the object. A change of a property may significantly impact the meaning or presentation of an object. Certain properties (for example, <pref>aria-multiline</pref>) are less likely to change than <a class="termref" href="#dfn-state">states</a>, but note that the frequency of change difference is not a rule. A few properties, such as <pref>aria-activedescendant</pref>, <pref>aria-valuenow</pref>, and <pref>aria-valuetext</pref> are expected to change often. See <a class="specref" href="#statevsprop">clarification of states versus properties</a>.</p>
+			  <p>[=attribute|attributes=] that are essential to the nature of a given <a>object</a>, or that represent a data value associated with the object. A change of a property may significantly impact the meaning or presentation of an object. Certain properties (for example, <pref>aria-multiline</pref>) are less likely to change than <a class="termref" href="#dfn-state">states</a>, but note that the frequency of change difference is not a rule. A few properties, such as <pref>aria-activedescendant</pref>, <pref>aria-valuenow</pref>, and <pref>aria-valuetext</pref> are expected to change often. See <a class="specref" href="#statevsprop">clarification of states versus properties</a>.</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="relationship|relationships">Relationship</dfn></dt>
+			<dt><dfn data-lt="relationship|relationships">Relationship</dfn></dt>
 			<dd>
 			  <p>A connection between two distinct things. Relationships may be of various types to indicate which <a>object</a> labels another, controls another, etc.</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="role|roles">Role</dfn></dt>
+			<dt><dfn data-lt="role|roles">Role</dfn></dt>
 			<dd>
 			  <p>Main indicator of type. <!-- (removing, vague) The object's role is the class of <a class="termref" data-lt="object">objects</a> of which it is a member. --> This <a class="termref" data-lt="semantics">semantic</a> association allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.</p>
 			</dd>
-			<dt><dfn data-export="">Root <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> node</dfn></dt><!--Not used-->
+			<dt><dfn>Root <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> node</dfn></dt><!--Not used-->
 			<dd>
 			  <p> The primary element containing non-metadata content. In many languages, this is  the document element but in <abbr title="Hypertext Markup Language">HTML</abbr>, it is the <code>&lt;body&gt;</code>.</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="semantics|semantic|semantically">Semantics</dfn></dt>
+			<dt><dfn data-lt="semantics|semantic|semantically">Semantics</dfn></dt>
```

Comment:
Needs export

Reply:
Exporting. Maybe reconsider this one... it's seems to be mostly using the generally accepted definition of "semantics" per the English language.